scalastyle-config.xml

```diff
@@ -499,4 +499,9 @@ This file is divided into 3 sections:
       and URIs to and from String. If possible, please use SparkPath.
     ]]></customMessage>
   </check>
+
+  <check customId="URLConstructor" level="error" class="org.scalastyle.file.RegexChecker" enabled="true">
```

Comment:
Should this rule also be added to `dev/checkstyle.xml`?

Reply:
It's already added by this PR, here.

- https://github.com/apache/spark/blob/b1af69eb7c017717f2f74e72634ba8ffa3ce6e0d/dev/checkstyle.xml#L209-L212

Reply:
Got